LEGO1/mxdsaction.h

```diff
@@ -57,6 +57,8 @@ class MxDSAction : public MxDSObject
   inline char *GetExtraData() { return m_extraData; }
   inline MxU16 GetExtraLength() const { return m_extraLength; }
   inline MxLong GetStartTime() const { return m_startTime; }
+  inline MxS32 GetLoopCount() { return m_loopCount; }
+  inline void SetLoopCount(MxS32 m_loopCount) { this->m_loopCount = m_loopCount; }
```

Comment:
Minor issue, but the argument should be called `p_loopCount`

Reply:
I was just following the precedent set here:
https://github.com/isledecomp/isle/blob/bb1782af46509c5cfe7d88e56411b95f7c8f42fc/LEGO1/mxdsaction.h#L56C1-L56C1